platforms/hyperledger-fabric/configuration/roles/create/anchorpeer/tasks/nested_anchorpeer.yaml

```diff
@@ -2,21 +2,16 @@
 # This task initiates the nested anchorpeer role for each organization
 
 # Check or wait for the join channel job to complete
-- name: "Check join channel job is done"
-  k8s_facts:
-    kind: Job
+- include_role:
```

Comment:
In all these files, do not remove the task name, it is easier to track the task with a name. I see you have used the name in the actual function, which is fine. just check if putting a name here also is possible.